docker/ci/github/Dockerfile

```diff
@@ -16,6 +16,16 @@ ENV PATH "/root/.cargo/bin:/root/bin/:$PATH"
 RUN mkdir -p /opt/cargo/
 RUN mkdir -p /opt/git/
 
+#Since github actions may not run as root in the docker image, and it's behavior
+#is undocument, and all request for information about any invarients are not responded to....
+#So we may symlink this locations to whatever home directory gha creates, currently /github/home/
```

Comment:
Comments are much more readable if there is a space between the comment character and the comment itself.

Reply:
I'll follow up on the nits in the next pr (already ready to go).